src/DebuggerForm.cpp

```diff
@@ -1588,6 +1591,51 @@ void DebuggerForm::processMerge(const QString& message)
 	}
 }
 
+QByteArray DebuggerForm::saveCommands() const
+{
+	QByteArray output;
+
+	auto counter = 0;
+	for (auto &command : commands) {
+		auto [name, description, source, icon, _] = command;
+		if (counter > 0) output.append(':');
+		output.append(name.toUtf8().toBase64());
+		output.append(';');
+		output.append(description.toUtf8().toBase64());
+		output.append(';');
+		output.append(source.toUtf8().toBase64());
+		output.append(';');
+		output.append(icon.toUtf8().toBase64());
+		counter++;
```

Comment:
Very much a detail:
`counter` is only used to distinguish the first from the other elements. Maybe it's clearer to instead use `bool first = true;` `if (!first) ...` `first = false;`?